src/java.base/share/classes/java/lang/VirtualThread.java

```diff
@@ -213,8 +213,15 @@ protected void onPinned(Continuation.Pinned reason) {
         private static Runnable wrap(VirtualThread vthread, Runnable task) {
             return new Runnable() {
                 @Hidden
+                @JvmtiMountTransition
                 public void run() {
+                    // notify JVMTI
+                    vthread.notifyJvmtiStart();
+
                     vthread.run(task);
```

Comment:
This doesn't look right, it needs to use try-finally.

Reply:
The discussion on this is on the next comment.